packages/one-app-runner/README.md

```diff
@@ -0,0 +1,258 @@
+
+**Note:** If you are making changes to One App at the same time you are making changes to your Module(s) then you will still need to [clone One App, serve your module to it, and start it yourself](https://github.com/americanexpress/one-app).
+
+## 🤹‍ Usage
+
+Note that [Docker](https://www.docker.com/community-edition#/download) is required and must be installed before **one-app-runner** can be used.
+
+### Quick start
+
+In your Holocron Module directory:
+
+1. Install:
+```bash
+npm install --save-dev @americanexpress/one-app-runner
+```
+
+2. Add an `npm` start script and a runner config to either your module's `package.json`.
+
+##### package.json
+```json
+"scripts": {
+  "start": "one-app-runner"
+},
+"one-amex": {
+  "runner": {
+    "modules": ["."],
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "one-app-dev:5.0.0",
+    "parrotMiddleware": "./dev.middleware.js"
+  }
+}
+```
+
+In the runner config you will want to tell `one-app-runner` what module map and One App version to use, what your root module is, where your module is (current directory in this case), and optionally where your [Parrot](https://github.com/americanexpress/one-app/blob/master/docs/recipes/Mocking-API-Calls.md) middleware file is located.
+
+##  Command options reference
+
+**one-app-runner** can be configured via command options or a configuration object in `package.json`:
+
+**module-map-url**
+
+module map for One App to use to fetch modules. This option is required.
+
+sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json
+```
+
+Or in `package.json`
+
+```json
+"one-amex": {
+  "runner": {
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "dockerhub.com/oneamex/one-app-dev:5.0.0",
+    "rootModuleName": "frank-lloyd-root"
+  }
+}
+```
+
+**root-module-name**
+
+Name of the module to use as a [root module](https://github.com/americanexpress/one-app#the-root-module)
+for your One App instance. This argument is required unless the `docker-image` argument is provided.
+
+sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --root-module-name frank-lloyd-root --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json
+```
+
+Or in `package.json`
+
+```json
+"one-amex": {
+  "runner": {
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "one-app-dev:5.0.0",
+  }
+}
+```
+
+**docker-image**
+
+Docker image to use for One App. It allows you to specify what Docker image and tag to pull down for One App. This can be an
+image you have built and tagged locally but is not published to a Docker registry. Whatever value is
+provided is passed down to `docker pull` and `docker run`.
```

Comment:
```suggestion
image you have built and tagged locally but is not published to a Docker registry. The value
provided is passed down to `docker pull` and `docker run`.
```

Reply:
is this required? if it's not, does it default to latest published?

Reply:
It is required. We can add sane defaults later but can't yet until the One App team starts publishing images on every release.

Reply:
Fixed in last commit.